draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -379,6 +379,14 @@ are communicating can be discovered.
 
 Mitigation of this threat is beyond the scope of this proposal.
 
+### Absence of mDNS candidates
+
+A web page SHOULD NOT be able to learn any user-specific information from the
```

Comment:
Since it's directly relevant to the core procedure, this text might fit better there, i.e., the #gathering section.

Reply:
> An implementation may also detect that mDNS is not supported by the available network interfaces. The ICE agent may skip steps 3 and 4 and directly decide to not expose the host candidate.

We need to revise this paragraph in #gathering with this PR.

Reply:
Updated PR, this seems much better indeed